_data/people.yaml

```diff
@@ -1,49 +1,72 @@
-organizers:
-  - name: Kevin Kipp
-    picUrl: https://pbs.twimg.com/profile_images/931552869915926528/sBpw-pNh_400x400.jpg
-    email: kevin.kipp@gmail.com
-    twitter: kevin_kipp
-    github:
-    linkedin:
-  - name: Patrick Costanzo
-    picUrl: https://pbs.twimg.com/profile_images/1230181384641183747/SKvgTCZL_400x400.jpg
-    email: costanzo.patrick@gmail.com
-    twitter: pcostanz
-    github:
-    linkedin:
-emeriti:
-  - name: Joe McCann
-    picUrl: https://pbs.twimg.com/profile_images/846386052910637056/tWGT0XAz_400x400.jpg
-    email:
-    twitter: joemccann
-    github:
-    linkedin:
-  - name: Alex Sexton
-    picUrl: https://pbs.twimg.com/profile_images/1034325028026769408/hz_G2I3L_400x400.jpg
-    email:
-    twitter: slexaxton
-    github:
-    linkedin:
-  - name: Rebecca Murphey
-    picUrl: https://pbs.twimg.com/profile_images/504070124509229056/6t-MUDgL_400x400.jpeg
-    email: rmurphey@gmail.com
-    twitter: rmurphey
-    github:
-    linkedin:
-  - name: Lon Ingram
-    picUrl: https://pbs.twimg.com/profile_images/970161690233864192/rztEhZFR_400x400.jpg
-    email: lawnsea@gmail.com
-    twitter: lawnsea
-    github:
-  - name: Aaron Stacy
-    picUrl: https://pbs.twimg.com/profile_images/3281266042/5069b845017701c465760d25ba54b83c_400x400.jpeg
-    email:
-    twitter: aaronj1335
-    github:
-    linkedin:
-  - name: Andrew Levine
-    picUrl: https://pbs.twimg.com/profile_images/950842904771284992/pZwj7Nim_400x400.jpg
-    email:
-    twitter: drewml
-    github:
-    linkedin:
+jmccann:
+  name: Joe McCann
+  avatar: https://pbs.twimg.com/profile_images/846386052910637056/tWGT0XAz_400x400.jpg
+  email:
+  homepage:
+  twitter: joemccann
+  github:
+  linkedin:
+  status: emeriti
+asexton:
+  name: Alex Sexton
+  avatar: https://pbs.twimg.com/profile_images/1034325028026769408/hz_G2I3L_400x400.jpg
+  email:
+  homepage:
+  twitter: slexaxton
+  github:
+  linkedin:
+  status: emeriti
+rmurphey:
+  name: Rebecca Murphey
+  avatar: https://pbs.twimg.com/profile_images/504070124509229056/6t-MUDgL_400x400.jpeg
+  email: rmurphey@gmail.com
+  homepage: http://rmurphey.com
+  twitter: rmurphey
+  github:
+  linkedin:
+  status: emeriti
+lingram:
+  name: Lon Ingram
+  avatar: https://pbs.twimg.com/profile_images/970161690233864192/rztEhZFR_400x400.jpg
+  email: lawnsea@gmail.com
+  homepage:
+  twitter: lawnsea
+  github:
+  linkedin:
+  status: emeriti
+astacy:
```

Comment:
this is a pretty arbitrary bit of obsessive compulsion, but i'd prefer to either go with the full `firstnamelastname` or the person's chosen twitter/github handle.

Reply:
😸 OCD is a job requirement for this kind of work and now is the right time to consider the options.

FWIW, I started using Twitter handles, but trying to remember or lookup everyone's Twitter usernames became a PITA. So I got lazy and went with what many corporations typically do for their email: first letter of given name concatenated to full surname. I considered full given name but wanted to avoid given name issues such as what to use: `Bob`, `Rob`, `Robert`, etc. Conveniently, these peepKeys are also relatively short. As a last consideration, I also reckoned that everyone may not have a Twitter/GitHub username -- but this is a small list of AustinJS organizers who can be compelled into getting a Twitter account (to manage @AustinJS).

☝️ So that explains my thought process, but I am not attached to it. If y'all think Twitter handles are the right way to go, then I can make the change now relatively easily.

Reply:
:shipit: